pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -149,12 +151,20 @@ func (w *watcher) run(ctx context.Context, repo git.Repo, repoCfg config.PipedRe
 		commitMsg                  string
 		includedCfgs, excludedCfgs []string
 	)
+
+	// Get rule to parse template.
+	rules := map[string]string{}
+	for _, env := range os.Environ() {
```

Comment:
Sorry, I couldn't catch your intention.
You intended that templating variables such as `{.Value}` and `{.EventName}` are passed through env?
IMO, it's possible to use these variables without setting them as an env var because it's passed when registering events.

Reply:
I see.
I was thinking I should set variables other than value and eventName, but maybe only value and eventName would be better.

Reply:
In this context, I think it's enough that we support only what the user input.